### DISCUSSION TITLE
[4.0] Border-width input and select fields

### DISCUSSION THREAD

**Discussion by angieradtke** [joomla/joomla-cms#47133] [2. Cold Features]
I have spent the last few days building a really really big site with Joomla 4. 
In the process, I've noticed that I'm having trouble figuring things out. 
 I noticed particularly negatively that the border  around the input fields is too thin and the text in them is barely distinguishable from the label. The contents of the fields are more difficult to grasp. Especially when you skim the page.
Maybe I'm getting old and my eyes are not well enough anymore : -)
![joomla-borders](https://user-images.githubusercontent.com/1172403/125597628-ba3d1cb1-522f-487f-9222-81ee2a68dcae.png)

By adjusting the border-width it will be better, I think.

```
.custom-select, .form-select {
   .......
    border: 3px solid var(--atum-bg-dark-20);
}


.form-control {
 ....
    border: 3px solid var(--atum-bg-dark-20);
...
}
```

**Comment by chmst** [comment]
I have the same problem with checkboxes. They are very visible when checked but hard to find if unchecked.

![grafik](https://user-images.githubusercontent.com/1035262/125637631-ac40bd8f-ae0e-4f0f-bfe7-e643e6e66dcb.png)

**Comment by brianteeman** [comment]
personally speaking 3px is too much. 

However if its going to be changed then it needs to be changed in a lot more places than suggested in the initial post so that all inputs have the same border. eg choices lists, checkboxes etc

Also combination fields like image preview need to be looked at as just changing the input border looks odd with the button and preview not having the same border

![image](https://user-images.githubusercontent.com/1296369/126059373-191b0cf1-01c3-4bbc-bb4c-e594e10697cd.png)


For the checkboxes alone they  actually have a different border to the inputs so you might want to try it with the same border values

**Comment by drmenzelit** [comment]
@ciar4n can you please have a look at this?

**Comment by Krshivam25** [comment]
I tried border : 2px
![border](https://user-images.githubusercontent.com/49578630/130124376-3bb156d6-ebd7-472f-81a5-cbb93e0e69af.png)

**Comment by brianteeman** [comment]
> I have the same problem with checkboxes. They are very visible when checked but hard to find if unchecked.

@chmst instead of changing the border thickness we can change its colour. Can you see if this change is better for you?


```
.form-check-input {
  border: 1px solid rgba(0, 0, 0, 0.55);
}
```
